server/message_hooks.go

```diff
@@ -416,8 +418,13 @@ func (p *Plugin) SendChat(srcUser string, usersIDs []string, post *model.Post) (
 			continue
 		}
 
+		fileName, fileExtension := getExtension(fileInfo.Name)
+		if fileName == "" || fileExtension == "" {
```

Comment:
This can lead to not uploading files without extension what is a legit usage. I don't think not having file extension is a reason to ignore the file. I don't even think that not having a filename is a reason to ignore it.